charts/atlas-deployment/Chart.yaml

```diff
@@ -4,15 +4,15 @@ description: A Helm chart to manage Atlas resources with Atlas operator
 
 type: application
 
-version: 2.4.1
+version: 1.1.0
```

Comment:
On the last update, the version on bumped to the same `version` of the `appVersion`(AKO chart). I brought it back to the line